common/derivation-checker.js

```diff
@@ -0,0 +1,45 @@
+import PrivKeyWallet from './libs/wallet/privkey';
+import assert from 'assert';
+const { exec } = require('child_process');
+const ProgressBar = require('progress');
+
+// FIXME pick a less magic number
+const derivationRounds = 100;
+const bar = new ProgressBar(':percent :bar', { total: derivationRounds });
+
+async function testDerivation() {
+  const privKeyWallet = PrivKeyWallet.generate();
+  const command = `docker run -e key=${privKeyWallet.getPrivateKey()} dternyak/eth-priv-to-addr:1.0`;
```

Comment:
Tagged to 1.0, pull out as a variable to set for future upgrades? Or just tag as `latest`?